gabrielaEnache/src/main/java/collections/store/Order.java

```diff
@@ -0,0 +1,89 @@
+package collections.store;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+    private static long id = 0;
```

Comment:
The same as in Item class, please take a look on the comment above

Reply:
Fixed